src/controllers/quote.js

```diff
@@ -107,3 +108,9 @@ function validateAmounts (sourceAmount, destinationAmount) {
     }
   }
 }
+
+function validatePrecisionAndScale (precision, scale) {
+  if (precision && scale) return
+  if (!precision && !scale) return
```

Comment:
`precision` and `scale` here are from the query arguments, so they are `Strings`.